Gruntfile.js

```diff
@@ -412,9 +414,8 @@ module.exports = function(grunt) {
 
   grunt.registerTask("komanda-package", function(platforms) {
     try {
-    fs.mkdirSync("package/");
-    } catch(e) {
-    }
+      fs.mkdirSync("package/");
+    } catch(e) {}
```

Comment:
I've just reindent, I don't know why there is a try/catch here.

Maybe test if folder exists before try to create it.